featuretools/entityset/api.py

```diff
@@ -1,6 +1,6 @@
 # flake8: noqa
 from .deserialize import read_entityset
 from .entity import Entity
-from .entityset import EntitySet
+from .entityset import LTI_COLUMN_NAME, EntitySet
```

Comment:
Does this need to be here?

Reply:
probably not; I was trying to avoid two imports for `from featuretools.entityset import LTI_COLUMN_NAME, EntitySet` in `test_ww_es` but can just change it to `from featuretools.entityset.entityset import LTI_COLUMN_NAME, EntitySet`